sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -9,12 +9,72 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     this.listenTo(this.categories, 'sync change:unread_count', this.render);
   },
 
+  events: {
+    'click #login-submit': 'loginSubmit',
```

Comment:
It puts variables in the URL if I make it submit and does not if I make it click.